lib/checker_jobs/configuration.rb

```diff
@@ -30,12 +33,12 @@ def jobs_processor_module
     end
   end
 
-  def emails_backend_class
-    case emails_backend
+  def notifier_class(notifier)
+    case notifier
     when :action_mailer
-      CheckerJobs::EmailsBackends::ActionMailer
+      CheckerJobs::Notifiers::ActionMailer
```

Comment:
I'm thinking, **for the future**, about having a constant such as:

``` ruby
NOTIFIER_CLASSES = {
  action_mailer: "CheckerJobs::Notifiers::ActionMailer",
}
```

Then use it this way:

``` ruby
def notifier_class(notifier)
  notifier_class_name = NOTIFIER_CLASSES.fetch(notifier) do
    raise CheckerJobs::UnsupportedConfigurationOption.new(:notifier, notifier)
  end
  Object.const_get(notifier_class_name)
end
```

This would avoid the case statement and hopefully would still use the auto-loading (not so sure about that last part).